components/speedreader/speedreader_util.cc

```diff
@@ -198,6 +198,8 @@ void DistillPage(const GURL& url,
     return;
   }
 
+  speedreader_service->metrics()->RecordPageView();
```

Comment:
Distillation may fail, I think we need to move this call to the `SpeedreaderTabHelper::OnDistillComplete`, there we can check result equals `DistillationResult::kSuccess`